mne/decoding/tests/test_time_gen.py

```diff
@@ -246,18 +249,33 @@ def test_generalization_across_time():
 
     # Test that gets error if train on one dataset, test on another, and don't
     # specify appropriate cv:
+    gat = GeneralizationAcrossTime(cv=ShuffleSplit())
+    gat.fit(epochs)
     gat = GeneralizationAcrossTime()
     with warnings.catch_warnings(record=True):
         gat.fit(epochs)
 
     gat.predict(epochs)
-    assert_raises(IndexError, gat.predict, epochs[:10])
+    assert_raises(ValueError, gat.predict, epochs[:10])
 
     # TODO JRK: test GAT with non-exhaustive CV (eg. train on 80%, test on 10%)
 
     # Make CV with some empty train and test folds:
     # --- empty test fold(s) should warn when gat.predict()
-    gat.cv_.test_folds[gat.cv_.test_folds == 1] = 0
+
+    class adhoc_cv():
```

Comment:
Class names are always camel case. And this should be a private one in any case?